database/index.js

```diff
@@ -0,0 +1,19 @@
+const { Sequelize, DataTypes } = require('sequelize');
+const schema = require('./seed.js');
+const sequelize = new Sequelize('availability', 'root', 'sqlcraft', {
```

Comment:
Store your database credentials in environment variables

Reply:
thanks @manedurphy!! I just pushed changes to address your comments. Could you please review when you get a chance?